server/src/resolvers/mapping/index.ts

```diff
@@ -47,7 +47,10 @@ export const importMapping = async (
     case MAPPING_VERSION_6:
       return handleV6(prismaClient, sourceId, resources)
     case MAPPING_VERSION_7:
-      return handleV7(prismaClient, sourceId, resources)
+      const existingUsers = (await prismaClient.user.findMany()).map(
```

Comment:
that's a little bit barbaric, I was thinking of doing the query like this:
```
const authors = comments.map(c => c.author.email).filter(filterDuplicates)
const res = prisma.user.findMany({email: {in: authors}})
const missingUsers = authors.map(a => !res.includes(a))
if (missingUsers.length > 0) throw Error(`missing users ${missingUsers.join(', '}`)
```
which would avoid fetching all users at once